wordpress-mobile/AztecEditor-iOS
# Simplifies HTML parsing and serialization.

This PR:

- Tones down the use of the `Converter` protocol, since it's really not bringing the advantages I hoped it would.  We could eventually end up removing it entirely but that's outside the scope of this particular PR.
- Renames some classes to make them more intuitive:
  - `InHTMLConverter` -> `HTMLParser`
  - `OutHTMLConverter` -> `HTMLSerializer`
  - `NSAttributedStringToNodes` -> `AttributedStringParser`
  - `HTMLNodeToNSAttributedString` -> `AttributedStringSerializer`
- Relocates several files.

**How to test:**

Review the new hierarchy by opening the project and looking at it.
Make sure there aren't duplicated files.
Run the unit tests.

**Integration:**

I've tested and none of the changes in this PR affect our current WPiOS integration.
